tests/app/views/test_users.py

```diff
@@ -231,6 +233,52 @@ def test_creating_buyer_user_with_good_email_domain_succeeds(self):
         data = json.loads(response.get_data())['users']
         assert data['active']
 
+    def test_creating_buyer_user_with_no_phone_number_succeeds(self):
+        response = self.client.post(
+            '/users',
+            data=json.dumps({
+                'users': {
+                    'emailAddress': 'joeblogs@digital.cabinet-office.gov.uk',
+                    'phoneNumber': '',
+                    'password': '1234567890',
+                    'role': 'buyer',
+                    'name': 'joe bloggs'}}),
+            content_type='application/json')
+
+        assert response.status_code == 201
+        data = json.loads(response.get_data())['users']
+        assert data['active']
+
+    def test_creating_buyer_user_with_bad_phone_number_fails(self):
+        response = self.client.post(
+            '/users',
+            data=json.dumps({
+                'users': {
+                    'emailAddress': 'joeblogs@digital.cabinet-office.gov.uk',
+                    'phoneNumber': '123456',
+                    'password': '1234567890',
+                    'role': 'buyer',
+                    'name': 'joe bloggs'}}),
+            content_type='application/json')
+
+        assert response.status_code == 400
```

Comment:
`u "JSON was not a valid format. u'123456' does not match u'^$|^\\\\+?([\\\\d\\\\s()-]){9,20}$'"`